includes/ets_user_qa_function.php

```diff
@@ -304,29 +321,32 @@ public function ets_ask_qustion_tab() {
 					<div class="table-responsive my-table">
 					<table class="table table-striped"> 
 					<?php
-					foreach ($etsGetQuestion as $key=>$value) {
-						?> 
-						<tr class="ets-question-top">
-								<td class="ets-question-title"><p><?php echo __('Question','ets_q_n_a'); ?>:</p></td>
-								<td class="ets-question-description"><p><?php echo $value['question'];?></p></td> 
-								<td class="ets-cont-right"><h6 class="user-name"><?php echo $value['user_name'] . "<br>";    
-								echo ($value['date']);
-								?></h6></td>
-						</tr>
+					foreach ($etsGetQuestion as $key => $value) {
+						if((isset($value['approve']) && $value['approve'] == 'yes') || !isset($value['approve'])){
```

Comment:
Do we need this condition again? we have filtered the approved questions already.

Reply:
yes no need i have removed